content/blog/2020-03-roxygen2-7.1.0/index.Rmarkdown

```diff
@@ -0,0 +1,113 @@
+---
+title: roxygen2 7.0.0
+author: Gábor Csárdi
+date: '2020-03-11'
+slug: roxygen2-7-1-0
+categories:
+  - package
+description: >
+  A minor roxygen2 release with some new features
+tags:
+  - r-lib
+  - devtools
+  - roxygen2
+photo:
+  url: https://pixabay.com/photos/fabric-wool-textile-texture-1031932/
+  author: John Perrett
+---
+
+We're chuffed to announce the release of [roxygen2 7.1.0](https://roxygen2.r-lib.org).
+roxygen2 allows you to write specially formatted R comments that generate R documentation files (`man/*.Rd`) and the `NAMESPACE` file.
+roxygen2 is used by over 8,800 CRAN packages.
+
+Install the latest version of roxygen2 with:
+
+```{r, eval = FALSE}
+install.packages("roxygen2")
+```
+
+This is a minor release, with many small improvements and bug fixes, and some new features.
+This blog post discusses the most important changes. See the [full changelog](https://roxygen2.r-lib.org/news/index.html#roxygen2-7-1-0) for all changes.
+
+## roxygen2 meets knitr
+
+We have been gradually improving roxygen2's markdown support, and this release has an exciting new markdown feature as well.
+Starting from roxygen2 7.1.0 you can write inline R code and markdown code fences directly in roxygen2 comments!
+roxygen2 runs the R code and insert its output into the documentation.
+
+### Inline R code
+
+For running R code inline, use the usual markdown backticks, and prefix the code with `r` and a space character.
+roxygen2 interprets the rest of the text within backticks as R code and evaluate it, and replace the backtick expression with its value.
+After all such substitutions, the text of the whole tag is interpreted as markdown, as usual.
+
+For example, the following inserts the date and the R version of the roxygen2 run.
```

Comment:
```suggestion
For example, the following code inserts the date and the R version of the roxygen2 run.
```

Reply:
One of those weird things where because "inserts" is also a noun it's a mushy sentence to read.